.github/workflows/glow-build.yml

```diff
@@ -29,12 +29,15 @@ jobs:
         fi
```

Comment:
You can add an argument called `docker-image` under `with:` and pass any public docker image. this should let you run everything in ubuntu.